tests/integration/pilot/mcs/discoverability/discoverability_test.go

```diff
@@ -74,8 +74,8 @@ var (
 
 	hostTypes = []hostType{hostTypeClusterSetLocal, hostTypeClusterLocal}
 
-	serviceA = match.ServiceName(echo.NamespacedName{Name: common.ServiceA, Namespace: echos.Namespace})
-	serviceB = match.ServiceName(echo.NamespacedName{Name: common.ServiceB, Namespace: echos.Namespace})
```

Comment:
`echos.Namespace` was empty at this point, so serviceA and serviceB matchers were returning 0 services and no test was executed.